lambdas/indexer/app.py

```diff
@@ -458,3 +459,23 @@ def nudge(event: chalice.app.CloudWatchEvent):
         log.info('Nudging queue with %i token(s) for a total value of %i', len(tokens), num_documents)
         for batch in chunked(tokens, 10):
             _send_tokens(batch)
+
+
+@app.schedule('rate(5 minutes)')
+def retrieve_fail_messages(event: chalice.app.CloudWatchEvent):
```

Comment:
I followed the same pattern, but it seems like you can rid off the warning by setting the parameter to `event_`